pyaedt/generic/general_methods.py

```diff
@@ -1574,6 +1574,84 @@ def _check_installed_version(install_path, long_version):
     return False
 
 
+class Help:  # pragma: no cover
+    def __init__(self):
+        self._base_path = "https://aedt.docs.pyansys.com/version/stable"
+        self.browser = "default"
+
+    def _launch_ur(self, url):
+        import webbrowser
+
+        if self.browser != "default":
+            webbrowser.get(self.browser).open_new_tab(url)
+        else:
+            webbrowser.open_new_tab(url)
+
+    def search(self, keywords, app_name=None, search_in_examples_only=False):
+        """Search one or multiple keywords.
+
+        Parameters
+        ----------
+        keywords : str or list
+        app_name : str, optional
+            One of PyAEDT app like ``"Hfss"``, ``"Circui"``, ``"Icepak"`` or any other available app.
+        search_in_examples_only : bool, optional
+            Whether to search in Examples only or not.
+            The default is ``False``.
+        """
+        if isinstance(keywords, str):
+            keywords = [keywords]
+        if search_in_examples_only:
+            keywords.append("This example")
+        if app_name:
+            keywords.append(app_name)
+        url = self._base_path + "/search.html?q={}".format("+".join(keywords))
+        self._launch_ur(url)
+
+    def getting_started(self):
+        """Open Pyaedt User guide page."""
```

Comment:
```suggestion
        """Open the PyAEDT User guide page."""
```
Should this be the User guide page or the Getting started page?